src/components/Collection/SeriesTopPanel.tsx

```diff
@@ -81,7 +81,7 @@ const SeriesTopPanel = React.memo(({ series }: SeriesSidePanelProps) => {
           className="!h-[14.5rem]"
           transparent
         >
-          <div className="grid h-32 grid-cols-1 gap-x-[4.5rem] gap-y-2 overflow-y-auto pr-2 text-base font-normal 2xl:grid-cols-2 2xl:pr-0">
+          <div className="shoko-scrollbar grid h-32 grid-cols-1 gap-x-12 gap-y-2 overflow-y-auto pr-2 text-base font-normal 2xl:grid-cols-2 2xl:pr-0">
```

Comment:
why `shoko-scrollbar` here?

Reply:
without that it isn't the thin scroll bar when it overflows early due to zoom levels

like this: ![firefox_2024-08-22_06-44-11](https://github.com/user-attachments/assets/d82fbfc3-1442-45ab-a4ba-dd1537069ad6)